### PR TITLE
Initial ruby syntax rules

### PR DESCRIPTION
**Pull request by j-mcnally**
A few ideas discussed on slack for our ruby style guide.

### CODE REVIEW

**Comment by j-mcnally** [comment]
:+1:

**Comment by johnkoht** [comment]
:+1:

File: ruby.md [hunk 1]
```diff
@@ -1 +1,52 @@
...
+  end
+```
+
+
+
+## Inline vs Block Conditionals / Rescues
+
+If they only affect one code statement, even if that one line is broken into multiple lines, the conditional is placed in-line.
+
+If the conditional applies to multiple code statements, wrape the code in a conditional block.
```
Comment (andimal) [inline ruby.md:33]: ...

File: ruby.md [hunk 2]
```diff
@@ -1 +1,52 @@
...
+                  it is a very bad thing to do." 
+      } unless current_user.member.patient?
+  end
+```
+
+
+
+## Inline vs Block Conditionals / Rescues
+
+If they only affect one code statement, even if that one line is broken into multiple lines, the conditional is placed in-line.
```
Comment (andimal) [inline ruby.md]: *effect

Comment (andimal) [inline ruby.md:31]: I don't agree with this. If there is a conditional that applies to a block that is 4 lines long, it should be stated before reading through the whole block.

**Pull request closed without merging**